From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Alexander <protonull@protonmail.com>
Date: Sat, 13 May 2023 18:13:31 +0100
Subject: [PATCH] Deprecate legacy SkinCache class

Deprecated instead removed because I don't know whether this is used or not.

diff --git a/src/main/java/vg/civcraft/mc/civmodcore/util/SkinCache.java b/src/main/java/vg/civcraft/mc/civmodcore/util/SkinCache.java
index 1bc49b524b4a4ee01bdf72659f9055932749aadd..bd478374de2877e6a35dbeba87154b8dc9ad4b71 100644
--- a/src/main/java/vg/civcraft/mc/civmodcore/util/SkinCache.java
+++ b/src/main/java/vg/civcraft/mc/civmodcore/util/SkinCache.java
@@ -23,9 +23,11 @@ import org.bukkit.scheduler.BukkitTask;
 
 /**
  * @author caucow ( https://github.com/caucow )
- */
+ *
+ * @deprecated Use {@link vg.civcraft.mc.civmodcore.utilities.SkinCache} instead. */ // CivPatched
 // Maybe some day this class can be made more generic and accept an ExecutorService in a constructor or something.
 // For now it works. Ree.
+@Deprecated(forRemoval = true) // CivPatched
 public class SkinCache {
 
 	private final Plugin plugin;
@@ -55,6 +57,8 @@ public class SkinCache {
 	 * @param downloadThreads Number of threads to use to handle async loading.
 	 */
 	public SkinCache(Plugin plugin, int downloadThreads) {
+		vg.civcraft.mc.civmodcore.utilities.DeprecationUtils.printDeprecationWarning(); // CivPatched
+
 		this.plugin = plugin;
 
 		this.skinCache = CacheBuilder.newBuilder()
